kotlintest-assertions/src/main/kotlin/io/kotlintest/properties/Gen.kt

```diff
@@ -363,6 +380,33 @@ interface Gen<T> {
       override fun random(): Sequence<UUID> = generateSequence { UUID.randomUUID() }
     }
 
+    fun localDate(minYear: Int = 1, maxYear: Int = 2030): Gen<LocalDate> = object : Gen<LocalDate> {
+      override fun constants(): Iterable<LocalDate> = emptyList()
```

Comment:
Interesting dates to be constants: February 28th, February 29th, Jan 1st and Dec 31st